context.c

```diff
@@ -34,6 +34,10 @@ static const char xml_header[] = "<?xml version=\"1.0\" encoding=\"utf-8\"?>"
 "<!ATTLIST buffer-attribute name CDATA #REQUIRED>"
 "]>";
 
+const struct iio_context_params default_params = {
+	0,
+};
+
```

Comment:
Empty initializers is forbidden in C, I'm just making sure it will compile everywhere.
A getter would be better, yes. I can change that.

Reply:
@nunojsa Better now?

Reply:
> Empty initializers is forbidden in C, I'm just making sure it will compile everywhere.

I meant just having `const struct iio_context_params default_params;`. I'm fairly positive that on linux (with gcc) this will be initialized to 0. But I'm not sure if that is an assumption we can make everywhere. IIRC, the C standard just mandates that this should be initialized to it's default value (not really sure if we can assume this default is always 0). Disregard my comment...

> @nunojsa Better now?

:+1: